apps/codeforafrica/payload.config.ts

```diff
@@ -0,0 +1,28 @@
+import { buildConfig } from "payload/config";
+import Pages from "./src/payload/collections/Pages";
+import { CollectionConfig } from "payload/types";
+import dotenv from "dotenv";
+
+dotenv.config();
+dotenv.config({ path: "./.env.local" });
+
+const appURL = process?.env?.CFA_PAYLOAD_PUBLIC_APP_URL;
```

Comment:
```suggestion
const appURL = process?.env?.PAYLOAD_PUBLIC_APP_URL;
```

or `PAYLOAD_PUBLIC_CFA_APP_URL` if we must. `PAYLOAD_PUBLIC_` prefix is required.